### ISSUE TITLE
Carbs slow to show in app

### ISSUE THREAD

**Issue by JohnKitching** [nightscout/nightguard#187]
The two screenshots show AAPS and NightGuard 7 mins after entering 30g of carbs on the NightGuard watch app. 
As the carbs are in AAPS they must have gone via Nightscout. 
Any chance of reading a bit more frequently?

![image](https://user-images.githubusercontent.com/48366213/158148064-83c0256f-8f2e-4846-a798-60fa2d1dfd6c.png)
![image](https://user-images.githubusercontent.com/48366213/158148092-a0362f43-8ba1-4cd8-ab52-b0b3e8bee956.png)

**Comment by dhermanns** [comment]
Hi John! It should update every 5 minutes. Its a tradeoff between reducing traffic and getting data as soon as possible.

What would you like to have as an update frequency?

**Comment by JohnKitching** [comment]
Ah ok. Don't worry. I was assuming once something had been written it was then reread to ensure it had gone through. It's not that important.

**Comment by dhermanns** [comment]
ok

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FY3XHVQYRBPBVADK0EGAWB4S&k=6713387de76bd7529c43465f15d2c279&m=WUxwcT&r=RaxVS3S).
TASTERMONIAL
